.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,6 +38,8 @@ Fixes #
 If no, just write "NONE" in the release-note block below.
 If yes, a release note is required:
 Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
+
+For more information on release notes see: https://github.com/kubernetes/release/blob/master/docs/release-notes.md
```

Comment:
IMO this link is very release specific and https://git.k8s.io/community/contributors/guide/release-notes.md would be better to link here.

/cc @justaugustus @saschagrunert @cblecker

Reply:
Ah, I didn't even know about that link @nikhita. But I agree, it's better suited here.

Reply:
We already have the contributors guideline linked in the template (see point 5 above). I’d suggest in either moving it or keeping it as it currently is.

Reply:
Ah true, I missed that actually. My personal preference would be moving it down to the release note block, where the information is required. It's easy to miss information in the beginning (as happened to me), especially since the PR text field is quite small and I guess people are quick to scroll down. Also out of the six points above only two (2 & 5) refer to information that is immediately required for the contributor to fill out. I'd prefer having this information at the beginning of the block I need to fill out.

But just my 2 cents. Let me know how you want to proceed, if no change is needed I'll close the PR 🙂

Reply:
> I'd prefer having this information at the beginning of the block I need to fill out.

I agree -- it's easy to miss
It makes more sense next to the release-note.
We could have it in both places or use a sentence to point back up to the links.

Reply:
I tend to have it closer to the release notes section, too. :)

Reply:
IMO we can just remove point 5 from above and move it to the release note block. Wdyt? :)

I'd prefer not to duplicate, and having the link in the release note block would make it more visible. :+1:

Reply:
...and one small nit: could you update the link to use git.k8s.io instead? This would become https://git.k8s.io/community/contributors/guide/release-notes.md.

Reply:
Awesome, I'll make the changes & squash this evening!